src/math/rpy.hpp

```diff
@@ -26,14 +26,15 @@ namespace pinocchio
     template<typename Scalar>
     Eigen::Matrix<Scalar,3,3> rpyToMatrix(const Scalar r, const Scalar p, const Scalar y)
     {
-      return (Eigen::AngleAxisd(y, Eigen::Vector3d::UnitZ())
-              * Eigen::AngleAxisd(p, Eigen::Vector3d::UnitY())
-              * Eigen::AngleAxisd(r, Eigen::Vector3d::UnitX())
+      typedef Eigen::AngleAxis<Scalar> AngleAxis;
+      return (AngleAxis(y, Eigen::Vector3d::UnitZ())
```

Comment:
Same for Eigen::Vector3d, you should use the one from the type related to AngleAxis

Reply:
Damn! I really did this too quickly...

Reply:
Damn! I really did this too quickly...

Reply:
Fixed